### PR TITLE
translation guide for email template

### PR DESCRIPTION
**Pull request by speedy008**
## Purpose of this pull request

This pull request (PR) Contain a description about how translation can ne done in email template

## 

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_theory.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/translate_theory.html
- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/translations/translate_theory.html

whatsnew
Added a section and code example about using [email templates](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_theory.html) to do translations

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/frontend-dev-guide/translations/translate_theory.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ To add the email template strings to the dictionary, use the `{{trans}}` [direct
 
 Custom email templates [added using the Admin panel] are not stored in the file system, and their strings are not added to the dictionary.
 
+To ensure that your new string is added to the dictionary and translated, use the `trans` method when outputting a string in a [email template].
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/translations/translate_theory.md]:
```suggestion
To ensure that your new string is added to the dictionary and translated, use the `trans` method when outputting a string in an [email template]({{ site.mage2100url }}app/code/Magento/Customer/view/frontend/email/account_new.html).
```

File: guides/v2.1/frontend-dev-guide/translations/translate_theory.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ To add the email template strings to the dictionary, use the `{{trans}}` [direct
...
+
+For example:
+
+- When only string is added in the email template:
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/translations/translate_theory.md]:
```suggestion
- When only a string is added in the email template:
```

File: guides/v2.1/frontend-dev-guide/translations/translate_theory.md [hunk 3]
```diff
@@ -36,6 +36,22 @@ To add the email template strings to the dictionary, use the `{{trans}}` [direct
...
+
+    ```html
+    {{trans "Lorem Ipsum is simply dummy text of the printing"}}
+    ```
+
+- When only string is added with variable value in the email template:
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/translations/translate_theory.md]:
```suggestion
- When only a string is added with a variable value in the email template:
```

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
Thanks @speedy008!

**Pull request merged**